windows/ReactTestApp/MainPage.cpp

```diff
@@ -76,4 +78,43 @@ namespace winrt::ReactTestApp::implementation
         ReactRootView().ComponentName(s.as<ComponentViewModel>()->AppKey());
     }
 
+    void MainPage::OpenReactMenu(IInspectable const &, RoutedEventArgs)
+    {
+        ReactButton().Flyout().ShowAt(ReactButton());
+    }
+
+    MenuFlyoutItem MainPage::GetComponentMenuButton(::ReactTestApp::Component &component)
```

Comment:
Nit: This can be a free function.

Reply:
This function accesses `this` when setting click listener to member function.

Reply:
Sorry, you're right 😄